Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst

```diff
@@ -88,7 +86,16 @@ The default storage uses :file:`fileadmin` unless this was configured
 differently, as explained in :ref:`fal-concepts-storages-drivers`.
 
 So, for this example, the resulting file path would typically be
-:file:`<document-root>/fileadmin/tmp/temporary_file_name.ext`
+:file:`<document-root>/fileadmin/final_file_name.ext`
+
+To store the file in a sub folder use :php:`$storage->getFolder()`::
+
+   $newFile = $storage->addFile(
+         '/tmp/temporary_file_name.ext',
+         $storage->getFolder('some_folder'),
```

Comment:
I think it would be useful to hint that `getFolder()` not simply accepts a folder name but an identifier, thus something like `some/nested/folder` also works. Otherwise people may think they need a chain of `$storage->getFolder(...)->getSubfolder(...)` calls.

Reply:
Sounds good, will adjust the PR